### PR TITLE
Dynamic Type support on AddCollabView

### PR DESCRIPTION
**Pull request by MarinaFX**
# Major Changes 

- [x] DT app review: EasyRancho has now officially full support for Dynamic Type users of iOS. AddCollaborator view has now full support for accessibility modes ranging from Extra Small to AccessibilityXXXL 🎉 
  - Final result: 
  - AddColaboratorView with Dynamic Type support AddColaboratorView with Dynamic Type support AddColaboratorView with Dynamic Type support

### CODE REVIEW

File: Superlista/SuperlistaApp.swift [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import SwiftUI
 import UIKit
 import CloudKit
-
```
Comment (gabizorzo) [inline Superlista/SuperlistaApp.swift:4]: why???

File: Superlista/Views/AddCollaboratorSheetView.swift [hunk 2]
```diff
@@ -33,138 +36,139 @@ struct AddCollaboratorSheetView: View {
...
+                    if let list = list, let user = dataService.user {
+                        if dataService.isOwner(of: list, userID: user.id) {
+                            Text("AddCollabTextTip")
+                                .font(.body)
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                                .padding(EdgeInsets(top: 8, leading: 20, bottom: 16, trailing: 16))
+                            
+                            Button(action: {
+                                self.showShareActionSheet.toggle()
+                                shareSheet(listID: list.id, option: "1", listName: list.title, ownerName: listOwner.name!)
```
[the remark this answers is not in the document]
Comment (gabizorzo) [reply Superlista/Views/AddCollaboratorSheetView.swift:52]: na verdade listOwner.name ?? getNickname() - aquela função pra pegar um random name

File: Superlista/Views/AddCollaboratorSheetView.swift [hunk 3]
```diff
@@ -33,138 +36,139 @@ struct AddCollaboratorSheetView: View {
...
-                            guard let list = list else {
-                                return
+                    }
+                    
+                    //MARK: AddCollaboratorSheetView List view
+                    if let collaborators = collaborators {
+                        if !collaborators.isEmpty {
+                            List {
+                                ForEach(0..<self.collaborators.count, id: \.self) { index in
+                                    CollaboratorListView(collaborators: self.$collaborators, list: list, name: collaborators[index].name!, index: index)
```
[the remark this answers is not in the document]
Comment (gabizorzo) [reply Superlista/Views/AddCollaboratorSheetView.swift:85]: na verdade collaborators[index].name ?? getNickname() - aquela função pra pegar um random name

File: Superlista/Views/AddCollaboratorSheetView.swift [hunk 4]
```diff
@@ -33,138 +36,139 @@ struct AddCollaboratorSheetView: View {
...
-                            Button {
-                                self.showCollabSheetView = false
-                            } label: {
-                                Text("AddCollaboratorTrailingNavigationLabel")
-                                    .font(.system(size: 17))
-                                    .bold()
-                                    .foregroundColor(.blue)
-                            }
+                        self.showShareActionSheet.toggle()
+                        shareSheet(listID: list.id, option: "2", listName: list.title, ownerName: listOwner.name!)
```
[the remark this answers is not in the document]
Comment (gabizorzo) [reply Superlista/Views/AddCollaboratorSheetView.swift:119]: na verdade listOwner.name ?? getNickname() - aquela função pra pegar um random name

**Review by gabizorzo — CHANGES_REQUESTED** [review]

**Review by gabizorzo — APPROVED** [review]

**Review by thaisrfernandes — APPROVED** [review]

**Pull request merged**